target/linux/ath79/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -146,10 +146,17 @@ case "$FIRMWARE" in
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
 		;;
 	tplink,archer-c58-v1|\
-	tplink,archer-c59-v1)
+	tplink,archer-c59-v1|\
+	tplink,archer-c6-v2)
 		ath10kcal_extract "art" 20480 12064
 		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
+
+		case "$board" in
+		tplink,archer-c6-v2)
+			ath10kcal_patch_mac $(macaddr_add $(cat /sys/class/net/eth0/address) -1)
```

Comment:
MAC address patching for Wave2 cards is done in etc/hotplug.d/ieee80211/10_fix_wifi_mac

Or does this not work for some reason?

Reply:
I moved it to etc/hotplug.d/ieee80211/10_fix_wifi_mac and it's okay. We have the same qca9886 in C58/C59/C6, so I guess this is the right place.

Reply:
I talked with @mkresin and 10_fix_wifi_mac should disappear in the mid-term. My assumptions regarding this point we outdated, i prepared a patch to sort this out for the C59. Sorry for the inconvenience.

https://patchwork.ozlabs.org/patch/1019488/